libs/styles/styles.css

```diff
@@ -86,31 +90,37 @@ header.global-navigation {
 h1 {
   font-size: var(--type-heading-xl-size);
   line-height: var(--type-heading-xl-lh);
+  scroll-margin-top: var(--feds-totalheight-nav);
```

Comment:
That's definitely better than a static value, though one Issue I forsee is when things are on top of the navigation.

There are already such requirements https://github.com/orgs/adobecom/discussions/1166

I don't think we necessarily need to fix this right now but we should keep it in mind, otherwise the scroll-location might be off in those cases.

Reply:
Can you explain a bit why the `scroll-margin-top` is needed and the top-offset isn't enough?
```
  window.scrollTo({
    top: topOffset - bufferHeight,
    behavior: 'smooth',
  });
```

Reply:
On safari, the scrollbar is automatically initiated even before the scrollToHashedElement is initiated, so when it first scrolls down, the nav bar hides the title. Also to resolve [MWPW-133331](https://jira.corp.adobe.com/browse/MWPW-133331), we need this CSS property.